pkg/controller/elasticsearch/license/apply.go

```diff
@@ -43,7 +43,7 @@ func applyLinkedLicense(
 	defer cancel()
 	current, err := updater.GetLicense(ctx)
 	if err != nil {
-		return err
+		return fmt.Errorf("while getting current license level %w", err)
```

Comment:
nittiest of nits: personal preference for `errors.wrap` here but obviously this is very much personal pref